pkg/landscape/sur/post.hoon

```diff
@@ -29,6 +29,7 @@
 +$  reference
   $%  [%graph group=resource =uid]
       [%group group=resource]
+      [%app =ship =desk path=@ta]
```

Comment:
Makes more sense as `[%app =ship =desk =path]` than throwing a `@ta` in there, no? @liam-fitzgerald @arthyn

Reply:
Yeah absolutely

Reply:
oh cool, yeah that makes sense, didn't know we had a type specifically for that